### PR TITLE
User: remove unused prop 'mfaSecret'

### PR DESCRIPTION
**Pull request by alxndrsn**
It looks like this prop was added speculatively in 2017 (938c75a43374878342593553ea324f53b90cbfcc), and has been unused since then.

### CODE REVIEW

File: lib/model/frames.js [hunk 1]
```diff
@@ -189,7 +189,7 @@ Submission.Attachment = class extends Frame.define(
 class User extends Frame.define(
   table('users'), aux(Actor),
   'actorId',                            'password',
-  'mfaSecret',                          'email',        readable, writable,
+  'email',        readable, writable,
```
Comment (alxndrsn) [inline lib/model/frames.js:192]: I'm not sure what the spacing here signifies, so this may no longer be correct.

Comment (matthew-white) [reply lib/model/frames.js:192]: I think it's meant to keep field attributes aligned across fields, sort of a column. This is the only field in the `User` frame with attributes, but I think that pattern is easier to see in other frames.

File: test/integration/api/audits.js [hunk 2]
```diff
@@ -55,7 +55,6 @@ describe('/audits', () => {
               audits[0].details.should.eql({ data: {
                 actorId: david.actor.id,
                 email: 'david@getodk.org',
-                mfaSecret: null,
```
Comment (matthew-white) [inline test/integration/api/audits.js:58]: I guess this means that `mfaSecret` will continue to exist in old audit log entries. I think that's fine though: it doesn't seem useful to scrub it from the audit log.

**Review by matthew-white — APPROVED** [review]

File: lib/model/frames.js (hunk 1 above)
Comment (alxndrsn) [reply lib/model/frames.js]: :+1: updated to match alignment in other frames

**Pull request merged**